src/overlays/actors/ovl_End_Title/z_end_title.c

```diff
@@ -115,7 +115,7 @@ void EndTitle_Draw(Actor* thisx, GlobalContext* globalCtx) {
  * This function is never executed in normal gameplay because actor params are never set to 1
  * Produces the same results as the main draw function, except without the title cards on the end screen
  */
-void func_80B65DA8(Actor* thisx, GlobalContext* globalCtx) {
+void EndTitle_DrawNintendoLogo(Actor* thisx, GlobalContext* globalCtx) {
```

Comment:
Idk if this should be named, the comment says this is unused so there may be a similar function that may make better use of the name

Reply:
nvm I thought EndTitle was much bigger than that, but this is the only function apart from the 4 main procs

also the comment is wrong, temple of time has an actor list with `ACTOR_END_TITLE` and params 1. Maybe the scene setup using it is unused but I would be surprised